basyx/aas/backend/couchdb.py

```diff
@@ -108,7 +108,7 @@ def _parse_source(cls, source: str) -> str:
 
     @classmethod
     def do_request(cls, url: str, method: str = "GET", additional_headers: Dict[str, str] = {},
-                   body: Optional[bytes] = None) -> Dict[str, Any]:
+                   body: Optional[bytes] = None) -> MutableMapping[str, Any]:
```

Comment:
Instead of changing the return type to the more generic `MutableMapping`we could also add `HTTPHeaderDict` as Union.

Reply:
I have no strong opinion on which type should be used here.